.husky/pre-commit

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+# generate graphql schema
+pnpm --filter backend generate
+# generate docs
+pnpm run docs
+# format
+pnpm run format
```

Comment:
This makes committing take so long I'm worried Travis or Matt will just end up re-removing the pre-commit hook...
Could we limit the formatting to just `docs/`?

Reply:
Yes, we should. I realized that, too. I'll do that on Monday.